LocalSettings.php

```diff
@@ -3640,6 +3648,9 @@
 			'editrestrictedtemplateprotected',
 			'editimportprotected',
 		],
+		'+documentcontrolwiki' => [
```

Comment:
The + is not needed here since it is already present in config name

Reply:
> The + is not needed here since it is already present in config name

@joritochip Oh, my understanding is that the `+` defines configuration settings that are in addition to the default configuration, no?

Reply:
@joritochip Did you mean the `+` sign for the `$wgAvailableRights` variable? If _so_, then yeah, I see the other wikis do not have that, but I'm not quite sure. Can you clarify that for me? I have also fixed this with this commit (https://github.com/miraheze/mw-config/pull/3838/commits/84565fcecf05a0d2e1ec586c0f56d028c109b19f)

Reply:
+ is used to append to default configuration, using + in default config appends the default key to the actual already set config is my understanding. I think it is needed here.